source/base/static/base/js/crawl.js

```diff
@@ -30,8 +30,9 @@ $( document ).ready(function() {
   }
 
   function onOffStatus(status){
-    onOffGroup(statuses.states[status]["disabled"], statuses.buttons, true);
```

Comment:
It was causing JavaScript errors in my console.  I *suspect* there's a state somewhere I need to set that wasn't on.